deploy/examples/oauth/README.md

```diff
@@ -3,8 +3,10 @@
 This example shows how to combine Grafana with the [OpenShift OAuthProxy](https://github.com/openshift/oauth-proxy). This will only work on OpenShift.
 
 ## Installation
-
-1. Create the [session secret](./session-secret.yaml) in the same namespace as Grafana.
-2. Create the additional [cluster role](./cluster_role.yaml) and [binding](./cluster_role_binding.yaml).
-3. Create the [config map](./ocp-injected-certs.yaml) in the same namespace as Grafana.
-4. Create Grafana from the [CR](./Grafana.yaml) in this example.
+1. Create your desired namespace, install grafana operator there and change `example-namespace` in all yaml files to your namespace.
+2. Create the [service account](./service_account.yaml) in the same namespace as Grafana.
+3. Create the [session secret](./session-secret.yaml) in the same namespace as Grafana.
+4. Create the additional [cluster role](./cluster_role.yaml) and [binding](./cluster_role_binding.yaml).
+5. Create the [config map](./ocp-injected-certs.yaml) in the same namespace as Grafana.
+6. Create Grafana from the [CR](./Grafana.yaml) in this example.
+7. Create the [route](./grafana_route.yaml) in the same namespace as Grafana.
```

Comment:
You're enabling route creation in your CR and also set the correct port and annotation. Why then create the route manually here?

Reply:
@obabec any update on this?

Reply:
I see, I'll give this a try with the operator created route. If that does not work, we can make sure to document why and also only use the custom route in this example.